SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -396,6 +397,8 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 			checkNoEntry();
 		}
 	}
+
+	bMapLoaded = true;
```

Comment:
nit: for clients we should really only be settings this to true if `GetWorld()->RemovePIEPrefix(GlobalStateManager->DeploymentMapURL) == GetWorld()->RemovePIEPrefix(GetWorld()->URL.Map)`

Reply:
In that case I guess this should be placed a few lines above where we request a controller.

Reply:
As discussed, not changing anything for now.